Documentation/CRDs/ceph-cluster-crd.md

```diff
@@ -1334,34 +1335,74 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
+   python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --cephfs-filesystem-name <filesystem-name> --rgw-endpoint  <rgw-endpoint> --namespace <namespace> --format bash
+   ```
+
+- `--namespace`: Namespace where CephCluster will run, for example `rook-ceph-external`
+- `--format bash`: The format of the output
+- `--rbd-data-pool-name`: The name of the RBD data pool
+- `--rbd-metadata-ec-pool-name`: (optional) Provides the name of erasure coded RBD metadata pool
+- `--rados-namespace`: (optional) Divides a pool into separate logical namespaces
+- `--cephfs-filesystem-name`: (optional) The name of the filesystem
+- `--cephfs-metadata-pool-name`: (optional) Provides the name of the cephfs metadata pool
+- `--cephfs-data-pool-name`: (optional) Provides the name of the CephFS data pool
+- `--rgw-endpoint`: (optional) The RADOS Gateway endpoint in the format `<IP>:<PORT>`
+- `--rgw-pool-prefix`: (optional) The prefix of the RGW pools. If not specified, the default prefix is `default`
+- `--rgw-tls-cert-path`: (optional) RADOS Gateway endpoint TLS certificate file path
+- `--rgw-skip-tls`: (optional) Ignore TLS certification validation when a self-signed certificate is provided (NOT RECOMMENDED)
+- `--monitoring-endpoint`: (optional) Ceph Manager prometheus exporter endpoints (comma separated list of <IP> entries of active and standby mgrs)
+- `--monitoring-endpoint-port`: (optional) Ceph Manager prometheus exporter port
+- `--ceph-conf`: (optional) Provide a Ceph conf file
+- `--cluster-name`: (optional) Ceph cluster name
+- `--output`: (optional) Output will be stored into the provided file
+- `--dry-run`: (optional) Prints the executed commands without running them
+- `--run-as-user`: (optional) Provides a user name to check the cluster's health status, must be prefixed by `client`.
+- `--restricted-auth-permission`: (optional) Restrict cephCSIKeyrings auth permissions to specific pools, and cluster. Mandatory flags that need to be set are `--rbd-data-pool-name`, and `--cluster-name`. `--cephfs-filesystem-name` flag can also be passed in case of CephFS user restriction, so it can restrict users to particular CephFS filesystem.
+
+  !!! note
+      Restricting the csi-users per pool, and per cluster will require creating new csi-users and new secrets for that csi-users, so it can only be used for new deployments.
+
+    Sample run:
+    ```console
+    python3 create-external-cluster-resources.py --cephfs-filesystem-name <filesystem-name> --rbd-data-pool-name <pool_name> --cluster-name <cluster-name> --restricted-auth-permission true --format <bash> --rgw-endpoint <rgw_endpoin> --namespace <rook-ceph-external>
+    ```
+
+- `--upgrade`: (optional) Upgrades the 'Ceph CSI keyrings (For example: client.csi-cephfs-provisioner) with new permissions needed for the new cluster version and older permission will still be applied.
+
+    Sample run:
+    ```console
+    python3 /etc/ceph/create-external-cluster-resources.py --upgrade
```

Comment:
I think the `/etc/ceph/` can/should be removed from the `create-external-cluster-resources.py` calls